MapboxCoreNavigation.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "MapboxCoreNavigation"
-  s.version = '2.2.0-alpha.2'
+  s.version = '2.2.0-beta.1'
```

Comment:
It's better to use `$ ./scripts/update-version.sh VERSION_NUMBER`, as it'll apply these changes automatically. Other than that everything else looks good.

Reply:
i changed it back to do so using update script.